### DISCUSSION TITLE
Error in fresh installation in AAPanel

### DISCUSSION THREAD

**Discussion by iamtinu** [kimai/kimai#3361] [On-Premise: Installation & Updates]
I am using aapanel which uses custom path for hosting and uses OpenLiteSpeed. OLS accepts default apache arugments and can perfectly run app for apache based env.
Default user permissions for folders are set for user www.

I am getting HTTP ERROR 500 on chrome and mozilla. what am I doing wrong. All php extensions are also installed.

[17/Jun/2022:15:25:07 +0000] "GET / HTTP/1.1" 500 0 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:101.0) Gecko/20100101 Firefox/101.0"'

If anyone needs temporary panel access I can provide for troubleshooting.

Please provide help. Thank You

**Comment by kevinpapst** [comment]
> I am using aapanel which uses custom path for hosting and uses OpenLiteSpeed

Never heard of that and cannot tell you if it might work.

Anyway: please provide logs. A 500 always causes error logs, unless those can't be written or are purposely deactivated.

**Reply by iamtinu** [reply]
*removed logs for sec. purpose

Happy to share logs.
Thank you for prompt reply

**Reply by iamtinu** [reply]
if these logs doesn't serve the purpose. I can clear and regenerate logs

**Reply by kevinpapst** [reply]
I expect a couple of lines directly posted here. Clear the log, trigger the error, copy & paste the logs.
And check the Kimai logs.

**Reply by iamtinu** [reply]
can you please tell me the location of kimai logs. no errors are being generated. only error I see is in access logs.
[17/Jun/2022:15:42:14 +0000] "GET / HTTP/1.1" 500 0 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:101.0) Gecko/20100101 Firefox/101.0"'

**Reply by iamtinu** [reply]
this is the conf file by aapanel

**Reply by iamtinu** [reply]
I really liked kimai2 and I want to use it badly.

**Reply by iamtinu** [reply]
here is the temporary login if you want to dive into panel. Its pretty clean and simple panel and every change is properly captioned in each area you browse.

Auto Expiration time
2022/06/17 23:56:00

**Reply by iamtinu** [reply]
I will remove the link after you explore or troubleshoot

**Reply by iamtinu** [reply]
Please let me know if you can think of some solution

**Reply by iamtinu** [reply]
got the logs now
[prod.log](https://github.com/kevinpapst/kimai2/files/8942137/prod.log)

**Reply by iamtinu** [reply]
Problem 1
    - Root composer.json requires PHP extension ext-xsl * but it is missing from your system. Install or enable PHP's xsl extension.
  Problem 2
    - lorenzo/pinky is locked to version 1.0.6 and an update of this package was not requested.
    - lorenzo/pinky 1.0.6 requires ext-xsl * -> it is missing from your system. Install or enable PHP's xsl extension.
  Problem 3
    - phpoffice/phpspreadsheet is locked to version 1.23.0 and an update of this package was not requested.
    - phpoffice/phpspreadsheet 1.23.0 requires ext-fileinfo * -> it is missing from your system. Install or enable PHP's fileinfo extension.
  Problem 4
    - lorenzo/pinky 1.0.6 requires ext-xsl * -> it is missing from your system. Install or enable PHP's xsl extension.
    - twig/inky-extra v3.4.0 requires lorenzo/pinky ^1.0.5 -> satisfiable by lorenzo/pinky[1.0.6].
    - twig/inky-extra is locked to version v3.4.0 and an update of this package was not requested.

To enable extensions, verify that they are enabled in your .ini files:
    - /www/server/php/74/etc/php.ini
You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.

**Reply by iamtinu** [reply]
I have all extensions installed
